cognitive/src/main/scala/com/microsoft/azure/synapse/ml/services/openai/OpenAIPrompt.scala

```diff
@@ -125,15 +125,19 @@ class OpenAIPrompt(override val uid: String) extends Transformer
     this, "apiType", "The OpenAI API type to use: 'chat_completions' or 'responses'",
     isValid = ParamValidators.inArray(Array("chat_completions", "responses")))
 
+  GlobalParams.registerParam(apiType, OpenAIApiTypeKey)
+
   def getApiType: String = $(apiType)
 
   def setApiType(value: String): this.type = set(apiType, value)
 
   val columnTypes = new StringStringMapParam(
     this, "columnTypes", "A map from column names to their types. Supported types are 'text' and 'path'.")
   private def validateColumnType(value: String) = {
-    require(value.equalsIgnoreCase("text") || value.equalsIgnoreCase("path"),
+    require(value == "text" || value == "path",
```

Comment:
nonblocking: since this is a breaking change for anyone previously passing 'TeXt', consider adding a warning for inputs that would have been accepted apart from casing differences